README.md

```diff
@@ -24,107 +24,107 @@ The CI environment supports both ARM and Bicep and can be leveraged using GitHub
 * For information on contributing, see [Contribution](<https://github.com/Azure/ResourceModules/wiki/Contribution%20guide>)
```

Comment:
@ArielRam99 don't forget to 'undo' this in a branch - not main. In main it will continue to update the table

Reply:
@MrMCake  I undid the change in a branch within my fork and merged that into main and it still shows 1 addition